### PR TITLE
Pass release name as an option to generate

### PR DESCRIPTION
**Pull request by anujdeshpande**
Change-type: major
Signed-off-by: Anuj Deshpande <anuj@balena.io>

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -67,11 +66,8 @@ const generate = async (
 		type: 'nodebuffer',
 		compression: 'DEFLATE',
 	});
-	fs.writeFileSync(
-		Path.join(options.output, `release_${time}.zip`),
-		data,
-		'binary',
-	);
+	const release = 'release_' + options.releaseName + '.zip';
```
Comment (stathismor) [inline src/index.ts]: Could use template literals (`) here

**Review by stathismor — COMMENTED** [review]
Looks good :+1:

File: src/index.ts (hunk 1 above)
Comment (anujdeshpande) [reply src/index.ts]: Done!

**Comment by anujdeshpande** [comment]
@balena-ci I self-certify!

**Review by ghost — APPROVED** [review]

**Pull request merged**